dev/integration_tests/flavors/macos/Runner/AppDelegate.swift

```diff
@@ -5,7 +5,7 @@
 import Cocoa
 import FlutterMacOS
 
-@NSApplicationMain
+@main
```

Comment:
@cbracken, this app appears to be busted before this migration.

Reply:
Oh interesting -- did someone disable the flavours test I wonder?

Reply:
It never passed: https://github.com/flutter/flutter/issues/119782
https://github.com/flutter/flutter/pull/125581

Reply:
That would do it then :)